From: Thomas Schwinge <thomas@schwinge.name>
Subject: [PATCH] check_native

2008-11-23  Thomas Schwinge  <tschwinge@gnu.org>

	* sysdeps/mach/hurd/check_native.c: New file.

For now only a dummy.

---

    make[2]: *** No rule to make target `/media/Stalin/tmp/glibc.hurd.gnu-2/inet/check_native.o', needed by `/media/Stalin/tmp/glibc.hurd.gnu-2/inet/stamp.o'.  Stop.
    make[2]: Leaving directory `/media/data/home/thomas/tmp/source/glibc/git/glibc.hurd/inet'
    make[1]: *** [inet/subdir_lib] Error 2

 sysdeps/mach/hurd/check_native.c |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/sysdeps/mach/hurd/check_native.c b/sysdeps/mach/hurd/check_native.c
new file mode 100644
index 0000000..8a829aa
--- /dev/null
+++ b/sysdeps/mach/hurd/check_native.c
@@ -0,0 +1,9 @@
+#include <stdint.h>
+
+void
+__check_native (uint32_t a1_index, int *a1_native,
+		uint32_t a2_index, int *a2_native)
+{
+#warning __check_native() not yet implemented
+  return;
+}
-- 
tg: (0234227..) t/check_native (depends on: baseline)
